professor2.spec

```diff
@@ -0,0 +1,16 @@
+### RPM external professor2 2.1.4
+## INITENV +PATH PYTHONPATH %i/lib/python`echo $PYTHON_VERSION | cut -d. -f 1,2`/site-packages
+Source: http://www.hepforge.org/archive/professor/Professor-%{realversion}.tar.gz
+
+Requires: py2-numpy py2-sympy py2-matplotlib root cython yoda eigen
+%prep
+%setup -n Professor-%{realversion}
+
+%build
+make CPPFLAGS=-I${EIGEN_ROOT}/include
+
+%install
+make install PREFIX=%{i}
```

Comment:
Does professor2 and professor provide the same binaries and shared libraries? If yes, then we might have an issue.